libsignal-protocol/src/store_context.rs

```diff
@@ -197,6 +197,19 @@ impl StoreContext {
         Ok(())
     }
 
+    /// Remove the session records corresponding to all devices of a recipientId
```

Comment:
Perhaps add a comment that this happens on address basis, so that it has to be called twice if you actually like to remove both the e164 and uuid versions.